### PR TITLE
Verify nuget package integrity

### PR DESCRIPTION
**Pull request by chcosta**
Fixes https://github.com/dotnet/arcade/issues/6435

Validation that this fixes aspnetcore official builds is still in progress.

### CODE REVIEW

**Comment by chcosta** [comment]
Validated, this resolves the package integrity issue in aspnetcore - https://dev.azure.com/dnceng/internal/_build/results?buildId=864782&view=results

**Comment by chcosta** [comment]
Oh, drat, I shouldn't have merged the change to target framework for the tests back. I'll fix the tests on Monday

File: src/Microsoft.DotNet.SignTool.Tests/SignToolTests.cs [hunk 1]
```diff
@@ -1116,14 +1116,31 @@ public void CheckPowershellSigning()
             });
         }
 
+#if NETFRAMEWORK
```
Comment (mmitche) [inline src/Microsoft.DotNet.SignTool.Tests/SignToolTests.cs:1119]: What's the meaning of the pre-processor directive here?

Comment (mmitche) [reply src/Microsoft.DotNet.SignTool.Tests/SignToolTests.cs:1119]: okay. Can you add a quick comment here for this, unless I missed it?

**Review by mmitche — APPROVED** [review]

**Pull request merged**